CHANGELOG.md

```diff
@@ -55,12 +58,12 @@
 - Dropped support for the `maud_lints` crate
   [66ddbca](https://github.com/lambda-fairy/maud/commit/66ddbcac986f099e309c28491c276de39340068a)
 - Update to rustc 1.31.0-nightly (77af31408 2018-10-11)
-    - The feature flags have changed again! Remove `#![feature(use_extern_macros)]` and `#![feature(proc_macro_non_items)]`, and add `#![feature(proc_macro_hygiene)]` in their place. See the [documentation][getting-started] for a working example.
+  - The feature flags have changed again! Remove `#![feature(use_extern_macros)]` and `#![feature(proc_macro_non_items)]`, and add `#![feature(proc_macro_hygiene)]` in their place. See the [documentation][getting-started] for a working example.
```

Comment:
These changes look unrelated. Can you please revert everything except the one changelog entry?

Reply:
done.